nemo_text_processing/inverse_text_normalization/fr/verbalizers/money.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+# Copyright 2015 and onwards Google, Inc.
```

Comment:
delete

Reply:
All? Or just google portion and add in affiliates?

Reply:
just google and add affi